.github/workflows/validate-agentic-docs.yml

```
@@ -0,0 +1,274 @@
+              stale_plans=$((stale_plans + 1))
+            fi
+          done < <(find agentic/exec-plans/active -name "*.md" -type f 2>/dev/null)
+
+          if [ "$stale_plans" -gt 3 ]; then
+            echo "Too many stale active exec-plans ($stale_plans). Clean up agentic/exec-plans/active/"
+            exit 1
+          fi
+          echo "Exec-plan freshness OK ($stale_plans stale)"
+
+      - name: Validate code paths referenced in docs
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2016:info:4:12: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)